pages/osx/lipo.md

```diff
@@ -0,0 +1,19 @@
+# lipo
+
+> Tool for handling Mach-O Universal Binaries.
+
```

Comment:
```suggestion
> More information: <https://ss64.com/osx/lipo.html>.

```
How about that one?

Reply:
Is putting links to online manpage viewers common in tldr? Then it kind of makes sense, that one, however, is really outdated. (look at the supported architectures: x86 and PPC, no ARM)

Reply:
We try to put a link to either the homapage or a man page in every tldr page. We`ve had to deal with a lot of outdated manpages already. I think it's still better than nothing.